### DISCUSSION TITLE
How to use Super CIC 12F629?

### DISCUSSION THREAD

**Discussion by StanMOI** [sanni/cartreader#989] [Need help]
Hello,

Need help, please.
Not really understand, once super cic programmed and installed, how to use this super CIC ?
Is there is some option in menu to flash CIC on PCB boards?
Thanks 
Nothing mentioned here:
[https://github.com/sanni/cartreader/wiki/Flashing-the-snesCIC](https://github.com/sanni/cartreader/wiki/Flashing-the-snesCIC)

**Comment by sanni** [comment]
With the snesCIC/superCIC installed you can dump SNES cartridges with SA1 special chip. Without snesCIC/superCIC installed you can't dump them.
